src/app/components/AssetSymbolRenderer/index.tsx

```diff
@@ -30,7 +30,7 @@ export function getAssetSymbol(asset: Asset) {
   if (symbolMap.hasOwnProperty(asset)) {
     return symbolMap[asset];
   }
-  return AssetsDictionary.get(asset).symbol;
+  return AssetsDictionary.get(asset)?.symbol;
```

Comment:
Maybe fallback to asset type itself in case it's not yet defined in dictionary?

```suggestion
  return AssetsDictionary.get(asset)?.symbol || asset;
```